packages/web3/src/TestGatingNFT.ts

```diff
@@ -278,3 +158,10 @@ export async function balanceOf(nftName: string, address: Address): Promise<numb
 
     return Number(balanceEncoded)
 }
+
+export const TestNFT = {
```

Comment:
Should this be TestERC721, to make room for TestERC1155, and MockERC20 be TestERC20?

Reply:
Seems reasonable